src/graphql/device/TypeDefs.js

```diff
@@ -43,15 +43,16 @@ type Device {
     devices: [HistoryDeviceInput]!
     dateFrom: String
     dateTo: String
+    operationType: Int
```

Comment:
At least add the list of possibles ID and their meaning, although the correct is an enum.

Reply:
Use `#describe#` on the previous line